hooks/price/types.go

```diff
@@ -1,10 +1,98 @@
 package price
 
-type Input struct {
+import (
+	"github.com/bandprotocol/chain/v2/pkg/obi"
+)
+
+const DefaultMultiplier = uint64(100000000)
+
+type CommonOutput struct {
+	Symbols    []string
+	Rates      []uint64
+	Multiplier uint64
+}
+
+type LegacyInput struct {
 	Symbols    []string `json:"symbols"`
 	Multiplier uint64   `json:"multiplier"`
 }
 
-type Output struct {
+type LegacyOutput struct {
 	Rates []uint64 `json:"rates"`
 }
+
+type Input struct {
+	Symbols            []string
+	MinimumSourceCount uint8
+}
+
+type Output struct {
+	Responses []Responses
+}
+
+type Responses struct {
+	Symbol       string
+	ResponseCode uint8
+	Rate         uint64
```

Comment:
why new OS not return multiplier na? @warittornc

Reply:
the multiplier is hardcoded into the oracle script, returning it is unnecessary